tooling/operation/get-parameters-as-json-schema.js

```diff
@@ -322,11 +328,20 @@ function constructSchema(data, prevSchemas = [], currentLocation = '') {
   ['allOf', 'anyOf', 'oneOf'].forEach(polyType => {
     if (polyType in schema && Array.isArray(schema[polyType])) {
       schema[polyType].forEach((item, idx) => {
-        schema[polyType][idx] = constructSchema(item, prevSchemas, `${currentLocation}/${idx}`);
+        schema[polyType][idx] = constructSchema(item, prevSchemas, `${currentLocation}/${idx}`, jwtDefaults);
       });
     }
   });
 
+  // Check to see if defaults and currentLocation exist and it's not just an empty object and blank
+  // If currentLocation is blank, jsonPointer returns the whole object so it will shove everything into formData
+  if (jwtDefaults && Object.keys(jwtDefaults).length > 0 && currentLocation) {
+    const userJwtDefault = jsonpointer.get(jwtDefaults, currentLocation);
```

Comment:
There's some cases where `jsonpointer.get()` will throw an exception so you should catch that here and just not add a default if it happens. Check out where I'm doing this in `searchForExampleByPointer`, there's some comments explaining why it might happen.